dockers/README.md

```diff
@@ -0,0 +1,91 @@
+# How to locally test
+First build the funcX web base image:
+```bash
+cd dockers/base
```

Comment:
Better to mention somewhere this is in `funcx-web-service` repo

Reply:
you're right, will move this to the top level of repo